simulator: none

topics: 
  cmd_vel_rviz: puzzlebot_sim1/cmd_vel
  odom_rviz: puzzlebot_sim1/odom
  cmd_vel_gazebo: puzzlebot_controller/cmd_vel
  odom_gazebo: puzzlebot_controller/odom
  cmd_vel_robot: /cmd_vel
  wr_robot: /wr
  wl_robot: /wl

parameters:
  Kp_dist: 0.3
  Ti_dist: 10000
  Td_dist: 0

  v_min: -15
  v_max: 15

  Kp_ang: 3.0
  Ti_ang: 10000
  Td_ang: 0

  w_min: -20
  w_max: 20